content/CORETutorials/JailsPluginsVMs/Plugins/MinIOPlugin.md

```diff
@@ -0,0 +1,208 @@
+Migrating from the built-in S3 service to the Minio plugin could require an extended service window.
+
+{{< enterprise >}}
+Beginning in CORE 13.0-U6, Enterprise customers with the S3 service running or enabled are prevented from upgrading to the next major version.
+
+TrueNAS Enterprise customers are strongly encouraged to contact iXsystems Support for assistance with the migration process.
+
+{{< expand "Contacting Support" "v" >}}
+{{< include file="content/_includes/iXsystemsSupportContact.md" >}}
+{{< /expand >}}
+{{< /enterprise >}}
+
+### Migration Overview
+Users with the S3 service enabled must upgrade CORE to version **13.0-U6** and complete the migration process **before** upgrading to a later major version.
+
+The migration path involves [installing a MinIO Client](#installing-the-minio-client) (`mc`) release with the required feature and function support to migrate the S3 service deployment.
+
+Next [configure the Minio plugin](#configuring-the-minio-plugin-for-migration) deployment and then [migrate data](#migrating-data) from the MinIO S3 service deployment to it.
+
+After migrating data, you can either create an archive of the TrueNAS S3 service or delete it.
+
+### Installing the MinIO Client
+
+The [MinIO Client](https://min.io/docs/minio/linux/reference/minio-mc.html) (`mc`) is command line tool with support for both filesystems and S3-compatible cloud storage services.
+Install `mc` to facilitate data the migration.
+Versions are available for Windows, Linux, and Mac.
+
+This tutorial uses the Windows Subsystem for Linux (WSL), but any standard Linux distribution follows the same process.
+Users should be familiar with WSL or the standard Linux CLI interface.
+
+{{< hint type=important >}}
+Installed MinIO client software version must be between **RELEASE.2022-06-26T18-51-48Z** and **RELEASE.2022-10-29T10-09-23Z**.
+Newer or older versions will not work properly.
+{{< /hint >}}
+
+To install the most recent compatible version of `mc`, enter:
+
+```
+curl https://dl.min.io/client/mc/release/linux-amd64/archive/mc.RELEASE.2022-10-29T10-09-23Z \
+  --create-dirs \
+  -o $HOME/minio-binaries/mc
+
+chmod +x $HOME/minio-binaries/mc
+export PATH=$PATH:$HOME/minio-binaries/
+```
+
+{{< trueimage src="images/CORE/13.0/MinioClientInstall.png" alt="MinIO Client Installed" id="MinIO Client Installed" >}}
+
+After installation completes, enter `mc --version` to confirm the compatible `mc` version is installed.
+
+### Configuring the Minio Plugin for Migration
+
+Due to incompatibility between versions, `mc` is unable to export configuration data from the S3 service.
+Manual configuration of the Mino plugin is needed to enable data migration.
+
+1. If you have not done so, configure the network ports for the Minio plugin.
+
+    a. Go to **Jails** and locate the jail matching the **Jail Name** entered during [installation of the Minio plugin](#installing-the-minio-plugin).
```

Comment:
in this instance, is **Jail Name** specifically referring to the UI field? it reads like a general reference to the jail name. We need to avoid mixing up mentions of the noun with the UI as it gets confusing. 
```suggestion
    a. Go to **Jails** and locate the jail matching the jail name entered during [installation of the MinIO plugin](#installing-the-minio-plugin).
```
You can use the UI field here but in the context of saying matching the name entered in **Jail Name** on the **Create Jail** screen. or something like this. don't mix up the noun and UI mentions.